src/bench/base58.cpp

```diff
@@ -13,7 +13,7 @@
 
 static void Base58Encode(benchmark::State& state)
 {
-    unsigned char buff[32] = {
```

Comment:
Now using `std::array<unsigned char, 32>`. Both `clang-tidy` and Clang Static Analyzer are now happy :-)